src/D2L.CodeStyle.Analyzers/Immutability/MutabilityInspector.cs

```diff
@@ -231,16 +228,7 @@ HashSet<ITypeSymbol> typeStack
 			}
 		}
 
-		public MutabilityInspectionResult InspectConcreteType(
-			ITypeSymbol type,
-			MutabilityInspectionFlags flags
-		) {
-			var typesInCurrentCycle = new HashSet<ITypeSymbol>();
-
-			return InspectConcreteType( type, typesInCurrentCycle, flags );
-		}
-
-		private MutabilityInspectionResult InspectConcreteType(
+		private MutabilityInspectionResult InspectConcreteTypeImpl(

```

Comment:
@saquibmian looks like the current idiom in MutabilityInspector is to suffix the private versions with `Impl` (rather than `Recursive`... maybe I changed that awhile ago.) Anyway, made the name change.